### PR TITLE
chore: fix - update recipe-hyprland.yml

### PR DESCRIPTION
**Pull request by b-wize**
Added a few additional install options from my previous hyprland config. These should help with base usability OOTB although will still require some configuration.

### CODE REVIEW

File: config/recipe-hyprland.yml [hunk 1]
```diff
@@ -28,6 +28,37 @@ modules:
...
+      # wayland utilities
+      - swaybg
+      - swayidle
+      - swaylock
+      - wlr-randr
+      - wl-sunset
+
+      # essentials for usability
+      - pipewire
+      - wireplumber
```
Comment (RoyalOughtness) [inline config/recipe-hyprland.yml]: there are many duplicates here that are already included in https://github.com/wayblueorg/wayblue/blob/live/config/common-packages.yml

Please remove those from this list

File: config/recipe-hyprland.yml [hunk 2]
```diff
@@ -28,6 +28,37 @@ modules:
       - hyprland
       - waybar
       - xdg-desktop-portal-hyprland
+      - xorg-x11-server-Xwayland
+
+      # wayland utilities
+      - swaybg
```
Comment (RoyalOughtness) [inline config/recipe-hyprland.yml]: should these be common packages?

File: config/recipe-hyprland.yml [hunk 3]
```diff
@@ -28,6 +28,37 @@ modules:
...
+      # improved desktop experience
+      - bluez
+      - bluez-tools
+      - blueman
+      - network-manager-applet
+      - pavucontrol
+
+      # additional GUI applications
+      - thunar
+      - lxappearance
```
Comment (RoyalOughtness) [inline config/recipe-hyprland.yml]: what is this used for?

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

File: config/recipe-hyprland.yml (hunk 3 above)
Comment (b-wize) [reply config/recipe-hyprland.yml]: lxappearance? Used for managing GTK themes, although I think it is only applying to GTK 2, but not 3 or 4. I was looking int nwg-look (https://github.com/nwg-piotr/nwg-look), but it currently isn't available on Fedora, yet.

Comment (RoyalOughtness) [reply config/recipe-hyprland.yml]: got it, let's keep it then

File: config/recipe-hyprland.yml (hunk 2 above)
Comment (b-wize) [reply config/recipe-hyprland.yml]: Looks like Xwayland is in common, but swaybg is currently not

**Review by b-wize — COMMENTED** [review]
Requested changes made, and comments have been added for the other applications to provide a short description of use case.

**Pull request closed without merging**